### DISCUSSION TITLE
FDS compilation with nfortran (NVIDIA HPC SDK)

### DISCUSSION THREAD

**Discussion by KingCZE** [firemodels/fds#13337] [General]
Hi, has anyone tried to compile FDS with nfortran (NVIDIA HPC SDK fortran compiler)? I would be curious about the makefile changes, compiler flags, and the final performance. (Obviously, there are many options, including the instruction sets, ACC, offloading to the GPU etc., so it would be hard to compare it objectively.)

**Comment by marcosvanella** [comment]
My experience is that nvfortran does not adhere to Fortran 2018 standard and fails to compile some FDS source files. This is an issue found in some compilers that points to vendors not being focused in Fortran as in C++. Same happens for languages and packages for offloading to accelerators.
